.gitignore

```diff
@@ -6,6 +6,5 @@
 
 *.gem
 .bundle
-Gemfile.lock
```

Comment:
It's good practice to have the lockfile committed to ensure repeatable builds

Reply:
if this is a gem is better to not include it as otherwise you'll lock down your consumers 😅